content/blog/2025/04/2025-04-28-election-announcement.md

```diff
@@ -0,0 +1,97 @@
+
+We’ll be doing our best to reach out to every constituent group to let them know they are eligible to nominate candidates for the election.
+
+**_If you are interested in nominating someone – or yourself – to be a candidate in this election, read this post in its entirety._**
+
+<!-- more -->
+
+## Election schedule
+
+Here is the timeline for this election season, which will happen in the second quarter of every year:
+
+* Announcement on Monday, April 28
+* Nominations from Saturday, May 3 to Friday, May 16
+* Campaigning from Saturday, May 17 to Friday, May 30
+* Voting from Saturday, May 31 to Friday, June 13
+* Results announced on Monday, June 16
+
+## How our elections work
+
+Our Governing Board is made up of 9 different constituency groups across 3 categories: nonprofit and community representatives, funder representatives, and foundation representatives. Each group has been allocated seats on the board, gets to nominate its own slate of candidates, and gets to vote within that slate of candidates. 
+
+For example, Individual Members get to nominate themselves, and vote on the Individual Members who are qualified candidates. The only exception is with the Managing Director (hi, that’s me) who automatically gets a seat on the Governing Board, but that’s just one seat of many.
+
+Here is the composition of seats on the Governing Board:
+
+|Community representatives|Funder representatives|Foundation representatives|
+|:---:|:---:|:---:|
+|4 Individual Members|4 Platinum Members|3 Guardians|
+|3 Ecosystem Members|3 Gold Members|2 Spec Core Team Members|
+|2 Associate Members|2 Silver Members|Managing Director|
+
+[Learn how to become a member](https://matrix.org/membership/) and [see a list of our current organizational members](https://matrix.org/support/).
+
+We will be seating candidates for half of the constituency groups in 2025, namely: Gold, Silver, Individual, and Associate Members.
+
+![A visual election timeline for the next few years: in 2024 we elect all constituencies, in 2025 we elect Gold, Silver, Individual, and Associate Members, in 2026 we elect Platinum, Ecosystem, Guardians, and Spec Core Team Members, in 2027 we elect Gold, Silver, Individual, and Associate Members, and so on.](/blog/img/2024-election-years.png)
+
+There are currently no term limits, but term limits are a best practice in open governance so we intend to introduce them as we find our collective footing.
+
+## Who can nominate
+
+You must be a member of one of the constituency groups in order to nominate yourself or someone else to be a candidate in the election. By far our largest constituency group are Individual Members, who number in the 100’s. If you have donated at least $60 USD to the Foundation since April 20, 2024 on Donorbox, then you are eligible to nominate and vote.
```

Comment:
> since April 20, 2024 on Donorbox

is this open ended, i.e. can I still decide to donate on June 7th to be able to vote?

Further, should we link here to said donorbox?

Reply:
Yeah, good to link to Donorbox. But no, a person cannot decide to donate on June 7th to donate so they can vote:

> We build our voter rolls on May 31st and can't change them after that!

Reply:
So the implication is May 31st, not e.g. also April 20.